src/model/application.rs

```diff
@@ -11,6 +11,7 @@ pub struct Configuration {
     pub general: General,
     pub administrators: Administrators,
     pub thread_introductions: Vec<Introduction>,
+    pub role_embed: RoleEmbed,
```

Comment:
This is missing in the current example json & schema.

Reply:
I know. I am going to add that tomorrow.